### ISSUE TITLE
Update Google Form

### ISSUE THREAD

**Issue by GeorgiaHCA** [alan-turing-institute/AutisticaCitizenScience#508]
## Summary

Link to example for proposed new Google Form: https://docs.google.com/forms/d/1A9WDJWl7MurySUA0-u20-WZWZxC25_mAjFris-0KKwU/viewform?edit_requested=true 

We need to update the existing Google Form so that it: 

a) takes into account LC's comments as well as additional input from the community
b) has a generic and flexible form so that we can get ethics approval to use it for asking a range of questions
c) is up-to-date with where the project currenly is 

## Date Due by

ASAP

## Details

We can't actually use this form until we have ethics approval 

## Still to be done: 

* [ ] Add self-describe options within check-box responses 
* [ ] Get feedback including user feedack
* [ ] Add icons or images to make it clearer

## Who can help

Anyone can help to review! In fact it would be really useful!

**Comment by GeorgiaHCA** [comment]
Heya @lottycoupat - I've created a Google Form which is designed to be an example format. I would like to apply for ethics using this example so that we can modify it as we progress to ask different questions. What do you think? Would be great to get your thoughts too when your back @KirstieJane. 

Would be great if you have a chance to have a look too @JamesSCTJ @skfantoni and @suzieqpid- I think it needs some visuals!

**Comment by lottycoupat** [comment]
@GeorgiaHCA might be a silly question but could you please confirm when this form would be used? Is it when someone uses the platform? Or when they partake in the project?

**Comment by GeorgiaHCA** [comment]
Good qusetion @lottycoupat! This is a survey for when people take part in the project. For instance we might want to use it to survey people about who we are missing out of research, or suggestions for accessibility, or opportunities, and so on. We've already collected data from a version of this however the question was very broad. I would like to make a more flexible version and ask for ethics approval updates on that basis.

**Comment by skfantoni** [comment]
I find the 'What is your connection to autism? Please select as many as apply' options a little confusing.

- I am autistic and I have an autism diagnosis
- I am autistic and I do not have an autism diagnosis
- I think that I might be autistic

I believe the third and second options are essentially the same. You may want to word it as 'I identify as autistic'. I just don't think anyone can factually state that they are autistic without a diagnosis.

**Comment by GeorgiaHCA** [comment]
This is an interesting point @skfantoni ! The reason for these options is that I'v heard some people say that they don't think diagnosis determines their autism (as it feels to them like an external label). I like your idea a "I identify as autistic" which is neutral - so we could have the options as: 


-     I identify as autistic and I have an autism diagnosis
-     I identify as autistic and I do not have an autism diagnosis
-     I think that I might be autistic

Would those options work for you?

**Comment by skfantoni** [comment]
@GeorgiaHCA I definitely feel more comfortable with that. I know diagnoses can be hard to get, I just fear that if we ‘allow’ people to bluntly state they have a condition without an actual diagnosis, we potentially open up the door to further misconception. Misdiagnosis is a big enough problem without allowing people to self-diagnose. For example, you and I have discussed potential mental health diagnoses for me but I wouldn’t put them in a form as I have no proof. This may just be my binary, autistic perspective of the situation but either way I think the new wording is better.

**Comment by skfantoni** [comment]
@GeorgiaHCA you may even want go as far as ‘ I don’t identify as autistic but have a diagnosis’
Denial is common, especially in regards to labels and not ‘ticking boxes’
